core/src/main/scala/org/apache/spark/internal/config/package.scala

```diff
@@ -1333,6 +1333,11 @@ package object config {
     .booleanConf
     .createWithDefault(true)
 
+  private[spark] val JAVA_SERIALIZER_CACHE_RESOLVED_CLASSES =
+    ConfigBuilder("spark.serializer.JavaSerializer.cache.resolvedClasses")
```

Comment:
Is `spark.serializer.javaSerializer.cache.resolvedClasses.` more following naming convention here?

Reply:
updated